dojoup/dojoup

```diff
@@ -76,14 +76,14 @@ main() {
   if [[ "$DOJOUP_REPO" == "dojoengine/dojo" && -z "$DOJOUP_BRANCH" && -z "$DOJOUP_COMMIT" ]]; then
     DOJOUP_VERSION=${DOJOUP_VERSION-stable}
     DOJOUP_TAG=$DOJOUP_VERSION
-    need_cmd jq
     
     # Normalize versions (handle channels, versions without v prefix
     if [[ "$DOJOUP_VERSION" == "stable" ]]; then
       # Fetch the list of releases from the GitHub API and get the first non-prerelease
       DOJOUP_TAG=$(curl -s "https://api.github.com/repos/${DOJOUP_REPO}/releases" \
```

Comment:
Maybe we should use [get the latest release API](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release)

Reply:
i thought about that but would include `vX.X.X-alpha` releases.